### ISSUE TITLE
Alt Text Needs Implementing and Poor Contrast

### ISSUE THREAD

**Issue by SaraKole** [Akash-Gupta-Sudo/Tic-Tac-Toe#1]
In order to create a more accessible website it would be helpful to implement the following two suggestions. 

1. Implement the use of alt text to support the use of  screen readers for people with visual impairments. According to the Web Content Accessibility Guidelines (WCAG) 2.2 guidelines not including alt text violates the WCAG 1.1.1 guideline. Without the use of alt text screen readers cannot provide any descriptions of images present for users who are visually impaired. Aa way to implement this for it to be helpful could be adding alt text for the X's and O's. 
2. There are two color contrast issues present. One being the "Reset!" button as well as “It’s Player X turn” message.  The current contrast ratio for these two elements are very low at a 2.81:1. WCAG 2.0 level AA requires a contrast ratio of at least 4.5:1 for normal text and 3:1 for large text.
3. Fix the skipped headers in the HTML format to ensure screen readers have a well structure when providing information and can help facilitate keyboard navigation.
